gpm_api/tests/test_io/test_checks.py

```diff
@@ -197,6 +239,33 @@ def test_check_version(
             checks.check_version(version)
 
 
+def test_check_product_version(
+    product_info: Dict[str, Any],
+    versions: List[int],
+) -> None:
+    """Test check_product_version()"""
+
+    # Check valid versions
+    for product, info in product_info.items():
```

Comment:
Maybe worth using `pytest.mark.parametrize`? So that the test units automatically list all problematic products ... and does not stop to first one?

Reply:
Yes I can do this, but then it counts as one test per product. So there will be several thousand tests reported by pytest. Is this fine? Or do you know a way to avoid this?

Reply:
will use [pytest-check](https://github.com/okken/pytest-check)